http/oidc/pom.xml

```diff
@@ -128,6 +128,12 @@
             <artifactId>keycloak-admin-client</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-services</artifactId>
+            <version>3.1.0.Final</version>
```

Comment:
@PrarthonaPaul Just a minor, please put a version as a property to the properties tag in the parent pom

Reply:
@PrarthonaPaul In the parent pom.xml we have properties tag where you can find versions of libraries, see [here](https://github.com/wildfly-security/wildfly-elytron/blob/2.x/pom.xml#L59). Place your keycloak-services dependency there. Then, as with other libraries in that parent pom.xml, place your dependency to the[ dependencyManagement tag](https://github.com/wildfly-security/wildfly-elytron/blob/2.x/pom.xml#L445) the same way as other deps are. Then you can just use this dependency in child modules without having to specify a version. So in http/oidc/pom.xml you will then be able to place it without a version tag

Reply:
Thank you! 
I have updated the pom files to reflect this.